layouts/_default/list.html

```diff
@@ -0,0 +1,26 @@
+{{ define "main" }}
+
+<div class="flex-l mt2 mw8 center">
+  <article class="center cf pv5 ph3 ph4-ns mw7">
+    <header>
+      <h1 class="f1">
+        {{ .Title }}
+      </h1>
+    </header>
+    <div class="nested-copy-line-height lh-copy f4 nested-links nested-img mid-gray">
+      {{ .Content }}
+    </div>
+  </article>
+</div>
+
+  <article class="pa3 pa4-ns nested-copy-line-height nested-img">
+    <section class="flex-ns flex-wrap justify-around mt5">
+      {{ range .Paginator.Pages }}
+        <div class="relative w-100 w-30-l mb4 bg-white">
+          {{- partial "summary.html" . -}}
```

Comment:
I'm getting deprecation warnings  on this line.
```
WARN 2022/08/31 17:38:03
Ananke Theme Warning:
You are currently using 'partial "summary"' in your project templates.
You should replace it with '.Render "summary"' as the use of this partial will be deprecated in future releases.
More info here: https://github.com/theNewDynamic/gohugo-theme-ananke/releases/tag/v2.8.1
```

```suggestion
          {{- .Render "summary" -}}
```

Reply:
Using which command did you get those warnings?

Reply:
`hugo serve`

Reply:
Ahh, it's because you have a different version of the ananke theme. Ill update it!

Reply:
also one reason I'm no fan of git submodules :P